phys/module_pbl_driver.F

```diff
@@ -2016,9 +2044,8 @@ SUBROUTINE pbl_driver(                                          &
 #endif
 
 
-   
-   IF (PRESENT(dtaux3d)) THEN
-       IF(gwd_opt .EQ. 1)THEN
+   IF (GWD_opt .EQ. 1) THEN
```

Comment:
@mdtoy @weiwangncar @dudhia 
GWD_opt is an optional variable. While NMM has this flag defined in the registry, this argument is not passed into the PBL scheme from module_PHYSICS_CALLS.F.

There needs to be an "IF(PRESENT(GWD_opt)) THEN" ahead of "IF (GWD_opt .EQ. 1) THEN". 

This is at least part of the NMM trouble.

Reply:
I updated module_pbl_driver.F per your suggestion.  Have pushed the change to Github.